docs/4.0/quickstart.md

```diff
@@ -163,6 +163,31 @@ new-node      xxxxx-xxxx-xxxx-xxxxxxx     10.0.10.2:3022
 !!! tip "NOTE": 
     Teleport also supports static pre-defined invitation tokens which can be set in the [configuration file](admin-guide.md#adding-nodes-to-the-cluster)
 
+## Adding a node located behind NAT
+With the current setup you've only been able to add nodes that have direct access to the 
+authentication server and within the internal IP range of the cluster. We recommend 
+setting up a [Trusted Cluster](../admin-guide/#trusted-clusters) if you have workloads split
+ across different networks / clouds. 
+
+Teleport Node Tunneling lets you add a node to an existing Teleport Cluster. This can be 
+useful for IoT applications or for managing a couple of servers in a different network.  
+
+Similar to [Adding Nodes to Cluster](/quickstart/#adding-nodes-to-cluster), use `tctl` to
+create a single-use token for a node, but this time you'll replace the auth-server IP with
```

Comment:
I think everywhere else we call it just "auth server", whatever we use, we should be consistent in the docs. Same for below.